package.json

```diff
@@ -9,6 +9,10 @@
     "type": "git",
     "url": "git@github.com:blablacar/ui-library.git"
   },
+  "engines": {
+    "node": "^12.16.1",
```

Comment:
Why not freeze the exact version ?

Reply:
Good question 👌
Because If I do, it will:

- Require direct and transitive dependencies that declare an `engines` key to have the exact same version.
- Require all applications using this library that declare an `engines` key to have the exact same version.

If these conditions are not meet, an "Unsupported engine for @blablacar/ui-library" will be thrown by npm during installation.